Traceback (most recent call last):
  File "/home/coders/Documentos/Felipe/PruebaDesempenoMod1.py", line 6, in <module>
    import tkinter as tk
ModuleNotFoundError: No module named 'tkinter'
